### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by rachelnabors**
- **adding GDPR compliance form example**
- **TF-1344 update fishtank README**
https://linear.app/tinyfish/issue/TF-1344/clean-up-fish-tank-documentationreadme-for-public-facing

### CODE REVIEW

**Comment by colriot** [comment]
I think this PR has more commits from other RPs @rachelnabors ?

File: examples/close_cookies/close_cookies.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        # Create a new AgentQL page instance in the browser for web interactions
+        page: Page = browser.new_page()
+
+        page.goto(URL)
+
+        # Use query_elements() method to fetch the cookies dialog button from the page
+        response = page.query_elements(QUERY)
+
+        # Check if there is a cookie-rejection button on the page
+        if response.cookies_form.reject_btn != None:
```
Comment (paveldudka) [inline examples/close_cookies/close_cookies.py:33]:
```suggestion
        if response.cookies_form.reject_btn:
```

**Review by paveldudka — APPROVED** [review]

File: examples/close_cookies/close_cookies.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+"""This example demonstrates how to close popup windows (like promotion form) with AgentQL."""
+
+import time
+
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# Set the URL to the desired website
+URL = "https://www.amazon.com/"
```
Comment (lozzle) [inline examples/close_cookies/close_cookies.py:9]: do we want to use something lighter for the examples?  i thought we're staying away from amazon

**Review by lozzle — APPROVED** [review]

File: examples/close_cookies/close_cookies.py (hunk 2 above)
Comment (rachelnabors) [reply examples/close_cookies/close_cookies.py:9]: Like scrapeme.live? Go for it.

**Pull request merged**